README.md

```diff
@@ -1880,16 +1876,17 @@ Any good resources on this?
 
 ## Taxes on Equity Compensation
 
-Now that we’ve covered the basic concepts of equity and taxes, we’ll get into some messy
-details of how they interact.
-
-As already discussed, employees can get restricted stock, stock options, or RSUs.
-The tax consequences for each of these is dramatically different.
+It is crucial to understand the tax implications of stock options and awards.
+Equity and taxes interact in complicated ways, and the tax consequences for an employee’s
```

Comment:
Language issue: It's tax consequences not "for" your equity but "of receiving" equity.

Reply:
good catch

Reply:
fixed ("for an employee receiving")

Reply:
The "for" is better in that context. The consequences are consequences for the employee. Without the employee, they don't exist. The "tax consequences of an employee receiving" could be consequences for anyone or anything. The old version made that mistake ("The tax consequences for each of these" should have been "of)